imports/ui/stylesheets/_variables.scss

```diff
@@ -2,6 +2,7 @@
 $primary-color: #008cba !default; // bondi blue
 $secondary-color: #cf515f !default; // chestnut rose
 $text-color: #333333 !default; // mine shaft
+$text-color-light: #ffffff !default; // white
 
 $background-color: #F4F5F8 !default; // athens grey

```

Comment:
wir arbeiten oft nach dem Prinzip: Lass weg was du nicht brauchst und wenn du glaubst du wirst es brauchen wirst du es meistens nicht. Du kannst es ja einfügen wenn wir es mal vorhaben es zu rebranden, bis dahin ist es denke ich zwecklos.

Warum: Meistens sowass dann einfach nur rauschen im code. Ästhetischer Code ist in meinen Augen einfach zu lesender Code. Du würdest auch keine Satzzeichen hinzufügen, obwohl man sie nicht braucht oder :) ?